common/network-types/src/session/utils.rs

```diff
@@ -26,19 +27,25 @@ impl RetryToken {
         }
     }
 
-    pub fn retry_in(&self, base: Duration, max_duration: Duration) -> Option<Duration> {
-        let duration = base.mul_f64(self.backoff_base.powi(self.num_retry as i32));
+    fn retry_in(&self, base: Duration, max_duration: Duration, jitter_dev: f64) -> Option<Duration> {
+        let jitter_coeff = if jitter_dev > 0.0 {
+            rand_distr::Normal::new(1.0, jitter_dev)
```

Comment:
What is the range on this value? Could it be 30004.23123?  Should the true range not be validated in the config?

Reply:
Good point. More then 4 Sigma has a negligible chance of happening, I'd set the max to 0.25